src/GitHubRepoManager/Functions/Public/Set-GitHubRepoVulnerabilitySetting.ps1

```diff
@@ -23,20 +25,27 @@ function Set-GitHubRepoVulnerabilitySetting {
         [Parameter(Mandatory = $true)]
         [String]$GitHubOrg,
         [Parameter(Mandatory = $true)]
-        [String]$RepoPrefix
+        [String]$RepoPrefix,
+        [Parameter(Mandatory = $false)]
+        [String]$ExcludedRepoPrefix
     )
 
     $Uri = "/orgs/$GitHubOrg/repos?type=all"
     Write-Verbose "Searching org $GitHubOrg"
     $Repos = Invoke-GitHubRestMethod -Method GET -Uri $uri
```

Comment:
Here we should be using Get-GitHubRepo in a similar manner to [Search-DotNetFramework](Search-DotNetFramework). This will allow for mocking of the response in a similar way to [this](https://github.com/SkillsFundingAgency/das-github-toolkit/blob/7a02cb3f6c9bc6d92da7b9bb747753f70f243a69/tests/UT.Search-DotNetFramework.Tests.ps1#L5) test so that we can test the increased complexity this change introduces

Reply:
Used Get-GitHubRepo